src/main/java/org/apache/cassandra/sidecar/config/yaml/RestoreJobConfigurationImpl.java

```diff
@@ -56,11 +55,12 @@ public class RestoreJobConfigurationImpl implements RestoreJobConfiguration
     @JsonProperty(value = "restore_job_tables_ttl_seconds")
     protected final long restoreJobTablesTtlSeconds;
 
-
-    @JsonProperty(value = RESTORE_JOB_LONG_RUNNING_HANDLER_THRESHOLD_SECONDS,
-    defaultValue = DEFAULT_RESTORE_JOB_LONG_RUNNING_HANDLER_THRESHOLD_SECONDS + "")
+    @JsonProperty(value = "restore_job_long_running_threshold_seconds")
```

Comment:
Why are we removing the constant to hardcode the string?

Reply:
It is mainly to remove the `defaultValue` declared in the annotation. The constants are removed to be consistent with other fields.